lib/parsers/json.js

```diff
@@ -775,6 +777,10 @@ class JoiJsonSchemaParser {
       delete schema.type
     }
   }
+
+  // _copyMetasToSchema is intended to be overridden by child parsers.
+  // eslint-disable-next-line no-unused-vars
+  _copyMetasToSchema(joiSpec, schema) { }
```

Comment:
What do you think if the `_copyMetasToSchema` implementation is here, and the `isKnownMetaKey` is the function required to be overridden by child parsers? 😃

Reply:
IMO this way provides more flexibility to child parsers if their needs on how meta is added ever diverges, although it creates a bit more code repetition. I'll add your suggested approach though.

Reply:
Done in 7b44c60